flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -683,4 +1058,8 @@ protected void checkEquals(CatalogView v1, CatalogView v2) {
 		assertEquals(v1.getOriginalQuery(), v2.getOriginalQuery());
 		assertEquals(v1.getExpandedQuery(), v2.getExpandedQuery());
 	}
+
+	protected void checkEquals(CatalogPartition expected, CatalogPartition actual) {
```

Comment:
we can just move `CatalogTestUtil.checkEquals(catalogpartitions)` to here and remove it from CatalogTestUtil